src/Employer/Employer.Web/Views/VacancyPreview/VacancyPreview.cshtml

```diff
@@ -35,6 +34,7 @@ public IHtmlContent EditLink(string routeName, VacancyPreviewSectionState state,
 <div class="grid-row">
     <div class="column-full">
         <h1 asp-show="@Model.CanShowDraftHeader" class="heading-xlarge small-bottom-margin">Complete your vacancy</h1>
+        <p class="small-top-margin">This is how your vacancy will appear to candidates on <a href="@Model.FindAnApprenticeshipUrl" rel="noopener" target="_blank">Find an Apprenticeship</a></p>
```

Comment:
For consistency it should be `Find an apprenticeship` lowercase `a` for `apprenticeship` like the other places on the site.

Reply:
👍